### PR TITLE
Added Build Node Status Badge

### PR DESCRIPTION
**Pull request by ysiddhapura**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,4 +107,6 @@ $ nodemon bin/www
 
 ## Contributing
 
-See the [Contributing](contributing.md) guide to contribute to the project!
\ No newline at end of file
+See the [Contributing](contributing.md) guide to contribute to the project!
+
+[![Build Node](https://github.com/ChicoState/PantryNode/actions/workflows/actions.yml/badge.svg)](https://github.com/ChicoState/PantryNode/actions/workflows/actions.yml)
```
Comment (Jooms) [inline README.md:112]: This should replace lines 22~26

**Review by Jooms — CHANGES_REQUESTED** [review]
Also, small request.

Can you move the second part of your PR title to the description?
Leaving only `Added Build Node Status Badge` as the title?

File: README.md (hunk 1 above)
Comment (ysiddhapura) [reply README.md]: Replaced Lines 22-26 with Github actions status badge

**Review by jayrevolinskyjr — APPROVED** [review]
Looks good to me

**Review by Jooms — APPROVED** [review]

**Pull request merged**